data/transactions/logic/ledger_test.go

```diff
@@ -607,6 +608,11 @@ func (l *Ledger) AppParams(appID basics.AppIndex) (basics.AppParams, basics.Addr
 	return basics.AppParams{}, basics.Address{}, fmt.Errorf("no app %d", appID)
 }
 
+// GenesisHash returns a phony genesis hash that can be tested against
+func (l *Ledger) GenesisHash() crypto.Digest {
+	return crypto.Digest{0x03, 0x02, 0x03}
+}
+
```

Comment:
nit: maybe to have a separate `testGenesis` global var in a test, and use in `TestGenHash` to embed this expected value into the teal code. Or just call `Ledger{}.GenesisHash()` - the `0x3, 02, 0x3 + 29 bzeros` was not immediate clean to me.

```suggestion
var testGenesis [32]byte =  [32]byte{0x03, 0x02, 0x03}

// GenesisHash returns a phony genesis hash that can be tested against
func (l *Ledger) GenesisHash() crypto.Digest {
	return crypto.Digest(testGenesis)
}
```

Reply:
Yeah, that's reasonable. I just have to marshall that testGenesis into the TEAL.

Reply:
done